Sources/IR/Function.swift

```diff
@@ -88,4 +103,369 @@ public struct Function {
     return result
   }
 
+
+  /// Appends to `self` a basic block in `scope` that accepts `parameters`, returning its address.
```

Comment:
```suggestion
  /// Appends a basic block in `scope` accepting `parameters`, returning its address.
```
Can avoid using `self`.  Everything is with respect to `self` unless otherwise stated.